test/rest.test.js

```diff
@@ -147,7 +149,7 @@ describe('strong-remoting-rest', function() {
         .end(expectErrorResponseContaining({ message: 'test-error' }, ['stack'], done));
     });
 
-    it('should disable stack trace', function(done) {
+    it.skip('should disable stack trace', function(done) {
```

Comment:
this test is for `process.env.NODE_ENV = 'production';` 
do we still want to support this?